src/pages/search/registration_search/RegistrationSearchResults.tsx

```diff
@@ -8,9 +8,16 @@ import { stringIncludesMathJax, typesetMathJax } from '../../../utils/mathJaxHel
 interface SearchResultsProps {
   searchResult: SearchResponse<Registration, RegistrationAggregations>;
   canEditRegistration?: boolean;
+  promotedPublications?: string[] | undefined;
```

Comment:
smør på flesk med både `?` og `undefined`. Husker vi snakket om denne og jeg mener `undefined` blir mest tydelig her, pga av at vi ønsker man skal sende inn noe, eller noe sånt... lenge siden merker jeg😅 
```suggestion
  promotedPublications: string[] | undefined;
```

Reply:
Om den ikke er optional, så må den også sendes inn på Landing Page. Vet ikke helt om det er en god løsning?

Reply:
da er det bedre å sette den som optional og fjerne undefined ja